src/components/maps/Marker.jsx

```diff
@@ -26,7 +26,7 @@ export default function Marker({
           style={{ backgroundColor: 'white', width: 'max-content' }}
         >
           <ButtonLink
-            href={`/sightings/${get(entry, 'guid')}`}
+            href={`/${get(entry, 'linkPath')}/${get(entry, 'guid')}`}
```

Comment:
let's use `entry?.linkPath` style when we can remember to do it!

Reply:
Done